conda.recipe/meta.yaml

```diff
@@ -109,3 +106,7 @@ extra:
     - anaconda-client
     - setuptools >=0.39
     - twine >=1.11
+  channels:
+    - bokeh
+    - conda-forge
+    - tethysplatform
```

Comment:
We would prefer not to use any extra channels if possible, thus in CI we install `channels` from pip. Perhaps you could go a long extra mile and submit `django-channels` feedstock to conda-forge (https://github.com/conda-forge/staged-recipes/issues/2405)? I'm pretty sure quite a few people would appreciate this.

Reply:
Ah are these channels actually read by the CI? That makes more sense.

Reply:
Yah there is no problem pip installing the few things that can be pip installed. I agree with @mattpap that we would like to avoid random anaconda.org channels. Sometime soon I am going to rework our `deps.py` script to generate `requirements.txt` and conda env files that can be checked in for ease of use. The conda env file can specify pip dependencies so dev setup will be even  more streamlined.

Reply:
I am giving a shot at adding these deps to conda-forge. https://github.com/conda-forge/staged-recipes/pull/11141

>Sometime soon I am going to rework our deps.py script to generate requirements.txt and conda env files that can be checked in for ease of use. The conda env file can specify pip dependencies so dev setup will be even more streamlined.

Just to confirm, my finding was that there aren't any pip-only dependencies in bokeh as of now, and no place to put them because of relying on `conda.recipe/meta.yaml`. I know conda's `environment.yml` format on the other hand does have a section for pip dependencies.

Reply:
There aren't any basic required package dependencies that are pip-only. But there are some "dev setup" or optional dependencies that might be, such as `channels`. It would be great to have `channels` on conda-forge, but we should accommodate setting a full dev setup running regardless, until it is. In any event affording conda env an `requirements.txt` will be a more familiar experience for people getting set up than what we have now,  think.